Bottom: 3dd7be5c77a838d1e52e783b5fc9390bbe9f74b3
Top:    69fc9430bd65433269275f6f2143c9e9e548b4f3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:36:30 +0100

Refresh of w-c-split-gathering-routines-f-0

---

diff --git a/src/draw.c b/src/draw.c
index d540287..75a6c8e 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -286,7 +286,7 @@ rat_callback (const BoxType * b, void *cl)
 }
 
 static void
-draw_element_package (element)
+draw_element_package (ElementType *element)
 {
   /* set color and draw lines, arcs, text and pins */
   if (doing_pinout || doing_assy)
